src/Build.OM.UnitTests/Construction/ProjectChooseElement_Tests.cs

```diff
@@ -268,13 +268,13 @@ public void ExcessivelyNestedChoose()
         public void SettingWhenConditionDirties()
         {
             string content = @"
-                    <Project xmlns='http://schemas.microsoft.com/developer/msbuild/2003' >
+                    <Project>
```

Comment:
💡 Make sure to verify there is still adequate coverage for cases where `xmlns` is included, since both approaches are supported

Reply:
Yup, I left it in a bunch of UTs, mostly ones which dealt with evaluation. It probably could be cleaned from even more UTs (many use `xmlns="msbuildnamespace"` instead of `xmlns="http://schemas.microsoft.com/developer/msbuild/2003">`), but whatever this change is large enough as-is.

Comparing repo-wide searches of "xmlns=" (which admittedly covers other namespaces eg in loc xml files):
Before: 2641 results in 377 files
After: 1693 results in 277 files

Reply:
This also had me worried but by inspection it looks ok (this concern is why I split out the "remove from unit tests" part of the change).